### PR TITLE
Experimental page creation

### PR DESCRIPTION
**Pull request by shinytoyrobots**
Wrote first draft of "About page" for the Experimental section

### CODE REVIEW

**Comment by shinytoyrobots** [comment]
@jeoffw - was this the kind of balance between "not production ready" and "more than just conceptual" you were thinking for how we define "Experimental"?

File: src/content/experimental/about/about.md [hunk 1]
```diff
@@ -3,3 +3,22 @@ label: Experimental
 title: About
 ---
 
+<page-intro>Experimental components, designs, and other resources are presented for testing and feedback. They are not intended for production use.</page-intro>
+
+## Introduction
+
+The Carbon team is continually updating components, and introducting new design and development guidelines into the Carbon Design System.
```
Comment (jeoffw) [inline src/content/experimental/about/about.md]: typo: introducting

File: src/content/experimental/about/about.md [hunk 2]
```diff
@@ -3,3 +3,22 @@ label: Experimental
...
+
+Experimental components, designs, patterns, and other resources are work that is not fully tested. But they are in a state where it can be productive for designers and developers to work with them.
+
+They are not intended for production use, and are subject to change. They may have some non-functional pieces, or be lacking in documentation. There is no guarantee of support.
+
+But these resources are also the future path of the Carbon Design System. As they are refined and improved, they may become core resources.
+
+## How can I help?
+
+Your feedback is critical for the success of the Carbon Design System, and for experimental components to improve and "graduate" to being production ready. Please open a Github issue from the footer of any page on the Carbon Design System website to let the team know what you think. Or, if you see problem, fix it and submit a pull request. We make sure that we acknowledge all contributors to the Carbon Design System.
```
Comment (jeoffw) [inline src/content/experimental/about/about.md]: typo - "if you see problem"

File: src/content/experimental/about/about.md (hunk 1 above)
Comment (shinytoyrobots) [reply src/content/experimental/about/about.md]: Fixed.

File: src/content/experimental/about/about.md (hunk 2 above)
Comment (shinytoyrobots) [reply src/content/experimental/about/about.md]: Fixed.

**Pull request merged**